zkevm-circuits/src/blake2b/circuit/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "blake2b_circuit"
```

Comment:
It is in a separate crate, because it has been written from scratch and not been integrated yet. I suppose that during integration it will be added as another module.

Reply:
Sounds good; we will integrate later on